src/module/system/action-macros/athletics/long-jump.ts

```diff
@@ -14,7 +14,7 @@ export function longJump(options: SkillActionOptions): void {
         traits: ["move"],
         event: options.event,
         callback: options.callback,
-        difficultyClass: options.difficultyClass,
+        difficultyClass: options.difficultyClass ?? { value: 15 },
```

Comment:
Why a default?

Reply:
That’s how it works now. It’s like High Jump, except with DC 15 instead of DC 30.

Reply:
> You Stride, then attempt a DC 15 Athletics check to make a long jump in the direction you were Striding. If you didn’t Stride at least 10 feet, you automatically fail your check. The GM might increase or decrease this DC depending on the situation.

Reply:
Lame!